src/ux/Display.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Interface that can be implemented by classes meant to display information directly to the user
+ * (for example, via {@code console.log()}).
+ * The interface/implementation pattern allows for extremely easy mocking of dependencies in tests.
+ */
+export interface Display {
```

Comment:
This paradigm is something @stephen-carter-at-sf developed for the Code Analyzer Team. It's easily mockable, which makes it really easy to test. If we ever (for some reason) need to add new methods, we can do that by putting them in the interface, and having the `UxDisplay` just invoke the corresponding method on the `Displayable`.

Reply:
I'll look over this tomorrow to see how I would use it in place of the `Display` system.